README.md

```diff
@@ -169,6 +169,25 @@ The date formatting will follow the format codes mentioned below, which are simi
 Any other character is printed as is. If you need to print the
 special characters (YMDymd), enclose them within quotes.
 
+#### List of Full name of month of MMM(Abbreviated month name )
+Below is the mapping of value of MMM, Abbreviation and name of full Month
+
+| Value of MMM | Abbreviation                  | Full Month Name |
+| ------------ | ----------------------------- | --------        |
+| 0            | Bai                           | Baishakh        |
+| 1            | Jes                           | Jestha          |
+| 2            | Asa                           | Aashadha        |
+| 3            | Shr                           | Shrawan         |
+| 4            | Bhd                           | Bhadra          |
+| 5            | Asw                           | Ashwin          |
+| 6            | Kar                           | Kartik          |
+| 7            | Man                           | Mangsir         |
+| 8            | Pou                           | Poush           |
+| 9            | Mag                           | Magh            |
+| 10           | Fal                           | Falgun          |
+| 11           | Cha                           |  Chaitra        |
+
+
```

Comment:
Let's not create a new heading for this table. I suggest it be part of **Formatting**. Here is my suggestion below.

```suggestion
For the list of month names and their abbreviations, you can refer to the table below.

| Value of Month | Abbreviation | Full Name |
| -------------- | ------------ | --------- |
| 0              | Bai          | Baishakh  |
| 1              | Jes          | Jestha    |
| 2              | Asa          | Aashadha  |
| 3              | Shr          | Shrawan   |
| 4              | Bhd          | Bhadra    |
| 5              | Asw          | Ashwin    |
| 6              | Kar          | Kartik    |
| 7              | Man          | Mangsir   |
| 8              | Pou          | Poush     |
| 9              | Mag          | Magh      |
| 10             | Fal          | Falgun    |
| 11             | Cha          | Chaitra   |

```